Simplenote/SimplenoteAppDelegate+Swift.swift

```diff
@@ -237,6 +238,24 @@ extension SimplenoteAppDelegate {
 
         SPTracker.trackSettingsStatusBarDisplayMode(hidden: Options.shared.statusBarHidden)
     }
+
+    @objc
+    func handleUserActivity(_ userActivity: NSUserActivity) -> Bool {
+        if userActivity.activityType == CSSearchableItemActionType {
+            presentNote(for: userActivity)
+            return true
+        }
+
+        return false
+    }
+
+    func presentNote(for userActivity: NSUserActivity) {
+        guard let uniqueIdentifier = userActivity.userInfo?[CSSearchableItemActivityIdentifier] as? String else {
```

Comment:
Nit: Hooow about if we just move the `simperiumKey` unwrapping to `handleUserActivity`? (that way we can just call `displayNote`?)